directives/row.py

```diff
@@ -0,0 +1,126 @@
+"""
+Directive for creating nested multirow columns with Bootstrap.
+
+Designed to be used with the point-of-interest directive.
+
+Options
+:width: sets column width. Maximum value 12.
+:column_class: Bootstrap classes can be applied. See example below.
+
+More about the Bootstrap classes https://getbootstrap.com/docs/4.4/layout/grid/#row-columns
+
+
+.. point-of-interest:: Test
+
+   .. row::
+
+     .. column::
+       :width: 8
+       :column_class: bg-warning text-center
+
+       .col-8 this is column's content.
+
+       .. row::
+
+         .. column::
+           :width: 6
+           :column_class: bg-light
+
+           .col-6
+
+         .. column::
+           :width: 6
+           :column_class: bg-secondary
+
+           .col-6
+
+     .. column::
+       :width: 4
+       :column_class: bg-success
+
+       .col-4
+
+"""
+
+from docutils.parsers.rst import Directive, directives
+from docutils import nodes
+
+import aplus_nodes
+
+
+class ColumnNode(nodes.General, nodes.Element):
+
+    def __init__(self, text):
+        super(ColumnNode, self).__init__()
+
+    @staticmethod
+    def visit_column(self, node):
+        self.body.append(self.starttag(node, 'div'))
+
+    @staticmethod
+    def depart_column(self, node=None):
+        self.body.append('</div>\n')
+
+
+class Column(Directive):
+    option_spec = {'width': directives.unchanged,
+                   'column-class': directives.unchanged,
+                   }
+
+    final_argument_whitespace = True
```

Comment:
Is `final_argument_whitespace = True` really needed? It looks like the directive does not use any arguments at all. If the default False value is OK, then you don't need to define it at all. 
https://www.sphinx-doc.org/en/master/extdev/markupapi.html#docutils.parsers.rst.Directive.final_argument_whitespace